homeassistant/components/yeelight/light.py

```diff
@@ -71,6 +70,7 @@
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_MINUTES = "minutes"
+ATTR_KELVIN = "kelvin"
```

Comment:
Shouldn't it be better to switch to ATTR_COLOR_TEMP_KELVIN, am I missing something or is this supposed to be a "transitional" step?

Reply:
This is for a custom service.
It is deprecated for the standard turn_on service, but in this case it was imported out of context for the custom service